npu.user.js

```diff
@@ -506,8 +507,11 @@ var npu = {
 					var select = $('<ul class="termSelect"></ul>');
 					var stored = npu.getUserData(null, null, ["termSelect", npu.getPage()]);
 					var found = false;
+					var admissionSemester = String($("#lblTrainingName").text().match(/:(\d{4}\/\d{2}\/\d)\[.*?\]\)/)[1]);

```

Comment:
Also, this will throw a `TypeError` if for whatever reason nothing is matched. Probably best to avoid that:

    var match = $("#lblTrainingName").text().match(/:(\d{4}\/\d{2}\/\d)\[.*?\]\)$/);
    var admissionSemester = match && String(match[1]);